vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -92,4 +92,14 @@ def textual_container_groups
     end
     h
   end
+
+  def textual_container_nodes
+    num_of_container_nodes = @record.number_of(:container_nodes)
+    label = ui_lookup(:tables => "container_nodes")
+    h = {:label => label, :image => "container_node", :value => num_of_container_nodes}
+    if  num_of_container_nodes > 0 && role_allows(:feature => "container_node_show")
+      h[:link] = url_for(:action => 'show', :controller => 'container_service', :display => 'container_nodes')
```

Comment:
The relationship here is many-1 (a container can run only on one node) so you have to show the single node (not a list of nodes).